src/uno/RoslynQuoter.Wasm/WasmScripts/AppManifest.js

```diff
@@ -1,7 +1,7 @@
 ﻿var UnoAppManifest = {
 
-    splashScreenImage: "Assets/SplashScreen.scale-200.png",
-    splashScreenColor: "#00f",
+	splashScreenImage: "Assets/SplashScreen.scale-200.png",

```

Comment:
here shouldn't have to specify the .scale-200   (unless it's not working in WebAssembly?)

Reply:
Tried removing it and the image doesn't show.